Sources/SwiftDocC/Model/Rendering/Symbol/AvailabilityRenderMetadataItem.swift

```diff
@@ -141,11 +141,8 @@ public struct AvailabilityRenderItem: Codable, Hashable, Equatable {
     }
 
     init?(_ availability: Metadata.Availability, current: PlatformVersion?) {
-        if availability.introduced == nil {
-            // FIXME: Deprecated/Beta markings need platform versions to display properly in Swift-DocC-Render (rdar://56897597)
-            // Fill in the appropriate values here when that's fixed (https://github.com/apple/swift-docc/issues/441)
-            return nil
-        }
+        // FIXME: Deprecated/Beta markings need platform versions to display properly in Swift-DocC-Render (rdar://56897597)
```

Comment:
@QuietMisdreavus have I understood correctly that this FIXME applies generally (and should be kept) or did it only imply when "introduced" was `nil` (which could never happen) in which case the FIXME can be removed?

Reply:
I think the comment still applies, since this is where the information will need to go when that's fixed.